From: Laurent Goderre <laurent.goderre@docker.com>
Subject: Fix mgr modules crashing when compiled with more modern tooling due to missing symbol.
diff --git a/src/pybind/mgr/mgr_module.py b/src/pybind/mgr/mgr_module.py
index 387936aadb7..3ce158cc508 100644
--- a/src/pybind/mgr/mgr_module.py
+++ b/src/pybind/mgr/mgr_module.py
@@ -1053,6 +1053,8 @@ class MgrModule(ceph_module.BaseMgrModule, MgrModuleLoggingMixin):
     BYTES = 0
     NONE = 1
 
+    NOTIFY_TYPES = []
+
     # Cluster log priorities
     class ClusterLogPrio(IntEnum):
         DEBUG = 0
